tutorials/AddingNewModel.md

```diff
@@ -0,0 +1,26 @@
+<h1>Adding New Model - Caffe</h1>
+
+1. For Setup instructions, look at the [README](https://github.com/Cloud-CV/Fabrik/blob/master/README.md).
+2. Open the <b>[Example folder](https://github.com/Cloud-CV/Fabrik/tree/master/example)</b>.
+3. Open the <b>[Caffe folder](https://github.com/Cloud-CV/Fabrik/tree/master/example/caffe)</b> and add the new model in the form of .prototxt file.
```

Comment:
What I earlier meant was to combine 2 and 3, and just say something like add the new model as a prototxt file to the `example/caffe` folder. Can you make that small change?